ddtrace/settings/_otel_remapper.py

```diff
@@ -0,0 +1,189 @@
+import os
+
+from ..constants import ENV_KEY
+from ..constants import VERSION_KEY
+from ..internal.logger import get_logger
+
+
+log = get_logger(__name__)
+
+
+OTEL_UNIFIED_TAG_MAPPINGS = {
+    "deployment.environment": ENV_KEY,
+    "service.name": "service",
+    "service.version": VERSION_KEY,
+}
+
+
+def _remap_otel_log_level(otel_value):
+    """Remaps the otel log level to ddtrace log level"""
+    if otel_value == "debug":
+        return "True"
+    elif otel_value == "info":
+        return "False"
```

Comment:
ddtrace-py only supports mapping `OTEL_LOG_LEVEL=debug` to `DD_TRACE_DEBUG=true`. All other log levels should not set `DD_TRACE_DEBUG`.

```suggestion
```